src/parsers/js.cr

```diff
@@ -30,17 +30,15 @@ module Mint
     end
 
     def not_interpolation_part(terminator : Char, stop_on_interpolation : Bool = true) : String?
-      # What characters to match
-      chars_to_match =
+      value =
         if stop_on_interpolation
-          "^#{terminator}#" # Until we find either a terminator or interpolation
```

Comment:
Doing this interpolation on every `not_interpolation_part` was very slow! Now there's no interpolation anymore :-)